manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -345,6 +349,10 @@ template: |
   - emptyDir:
       medium: Memory
     name: istio-envoy
+  {{- if eq .Values.global.proxy.enableCoreDump true }}
+  - name: coredumps
```

Comment:
this should be `istio-coredumps` or it will break if the user has any volume with the same name